ideainspect.groovy

```diff
@@ -242,15 +239,38 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
-private analyzeResult(File resultPath, List<String> acceptedLeves,
-                      List skipResults, List skipIssueFilesRegex) {
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println " "
+  println "#"
+  println "# Modifying idea.properties for scope: $cliOpts.sc"
+
+  def ideaHome = cliOpts.i ?: (System.getenv("IDEA_HOME") ?: "idea")
+  def libraryHome = cliOpts.li.replace("~", System.getProperty("user.home"))
+  def ideaPropsFile = new File(libraryHome + "/idea.properties")
+  def outputLines = new ArrayList<String>();
+  assertPath(ideaPropsFile, "Idea Properties File",
+              "Use the `lib` proeprty in `.ideainspect` or the `-li` command line option to point\n" +
+                      "to a valid preferences folder.")
+  def lines = ideaPropsFile.text.split(System.getProperty("line.separator"))
+  for(line in lines){
+    if(!line.contains("idea.analyze.scope")){
+      outputLines.add(line);
+    }
+  }
+  outputLines.add("idea.analyze.scope=" + cliOpts.sc)
+  ideaPropsFile.newWriter().withWriter { w ->
```

Comment:
How does this fail if the file is write protected (which I assume would be the default in a system-wide installation).

We should fail gracefully here and give the users a hint/note about that.